src/analysisd/rules.c

```diff
@@ -1136,6 +1140,44 @@ int Rules_OP_ReadRules(const char *rulefile)
                         }
 
                         free(s_norder);
+                    } else if (strcasecmp(rule_opt[k]->element, xml_mitre) == 0) {
+                        int i;
+                        int l;
+                        XML_NODE mitre_opt = NULL;
+                        mitre_opt = OS_GetElementsbyNode(&xml, rule_opt[k]);
+
+                        if (mitre_opt == NULL) {
+                            merror("Rule '%d' without any option. "
```

Comment:
This message is too generic. Please change it to something like: `Empty Mitre information for rule '%d'`